templates/default/sklearn_flask_server.py.erb

```diff
@@ -1,109 +0,0 @@
-"""
```

Comment:
I don't think you should remove this file. It's templated in install.rb:678 and used by `/srv/hops/sbin/sklearn_serving.sh`

Reply:
Oh true, I was grepping with the `.erb` extension and didn't find anything. Reverting, thanks.